### PR TITLE
Wait for code screen 

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-react/src/ui/Onboarding/EmailFlowCode.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, {useState} from 'react';
+
+export interface EmailFlowCodeProps {
+  onConfirm: () => void;
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/EmailFlowCode.tsx:4]: onConfirm: (code: string) => void

File: universal-login-react/src/ui/Onboarding/EmailFlowCode.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  onBack: () => void;
+  onCodeChange: () => void;
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/EmailFlowCode.tsx:6]: redundant onCodeChange: () => void

File: universal-login-react/src/ui/Onboarding/EmailFlowCode.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+export const EmailFlowCode = ({onConfirm, onCodeChange, onBack}: EmailFlowCodeProps) => {
+  const [code, setCode] = useState('');
+
+  <div>
+
```
Comment (xitronix) [inline universal-login-react/src/ui/Onboarding/EmailFlowCode.tsx:13]: add input

File: universal-login-react/src/ui/Onboarding/EmailFlowCodeScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+interface EmailFlowCodeScreenProps extends EmailFlowCodeProps{
+  title: 'Create account' | 'Log in';
+  hideModal: () => void;
+}
+
+export const EmailFlowCodeScreen = ({title, hideModal, onConfirm, onCodeChange, onBack}: EmailFlowCodeScreenProps) => (
+  <OnboardingStepsWrapper
+  title={title}
+  hideModal={hideModal}
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Onboarding/EmailFlowCodeScreen.tsx:13]: steps={4}
progress={2}

**Comment by JustynaBroniszewska** [comment]
Will be done in following PR

**Pull request closed without merging**